src/platform/web/ui/css/themes/element/theme.css

```diff
@@ -726,6 +726,9 @@ button.link {
     list-style: none;
     margin: 0;
     width: calc(100% - 48px);
+    display: flex;
```

Comment:
not sure why the delete button should be too wide? I'd rather not wrap the text buttons and keep one per line.

Reply:
This should be fixed now when you rebase your branch on master

Reply:
(See #509)